content/handbook/support/license-and-renewals/workflows/self-managed/trials.md

```
@@ -0,0 +1,129 @@
+
+### 緊急の週末ライセンス
+
+オンコール中で、ライセンスを生成する必要があるが CustomersDot インターフェイスにアクセスできない場合は、[週末の緊急対応 - ライセンスリクエスト](/handbook/support/license-and-renewals/workflows/self-managed/license_for_weekend_emergencies) ワークフローに従ってください。
+
+### SFDC で生成されるトライアルライセンス延長
+
+アカウントエグゼクティブ（AE）は、更新案件のクローズに想定以上の時間がかかっている場合に、SalesForce.com（SFDC）を使用して顧客に Self-managed の 21 日間ライセンス延長を発行できるようになりました。AE がこの機能を使用すると、ライセンスコードが自動的に生成され、L&R サポートの関与なしに顧客に送信されます。[一時的な更新延長](/handbook/product/groups/fulfillment/#temporary-renewal-extensions) のハンドブックエントリにこのアプローチが文書化されています。この機能の悪用を防ぐためのガードレールが設けられており、その結果、更新イベントごとに発行できるライセンス延長は 1 回のみです。そのため、L&R サポートがさらにライセンス延長を生成する必要がある場合があります。
+
+#### 手動の一時延長に対するセールス AE のリクエスト
+
+Self-managed のライセンス延長が必要な場合、AE はまず [SFDC のプロセス](#sfdc-generated-trial-license-extensions) を使用することが期待されます。ただし、延長期間が経過した場合、AE は [内部リクエストフォーム](https://support-super-form-gitlab-com-support-support-op-651f22e90ce6d7.gitlab.io/) を使用して L&R サポートに直接ライセンス延長をリクエストできます。**GitLab Support Internal Requests for Global customers** リクエストオプションと、内部リクエストの種類として **Extend an (almost) expired subscription** を使用します。このプロセスにはいくつかの注意点があり、以下に記載します。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)